package.json

```diff
@@ -17,7 +17,8 @@
     "@lhci/cli": "0.3.12",
     "@lhci/utils": "^0.3.12",
     "is-windows": "^1.0.2",
-    "lodash": "^4.17.15"
+    "lodash": "^4.17.15",
+    "puppeteer-core": "^2.1.1"
```

Comment:
Does it make sense to include `puppeteer-core` into the action?
I think the way `lhci` handles the dependency is better because it allows users to include any version and release new versions without depending on a major library.

Reply:
Sure, it can be in lhci.